dataservice/api/sequencing_experiment/schemas.py

```diff
@@ -90,6 +116,21 @@ class Meta(BaseSchema.Meta):
                                  validate=validate_positive_number)
     experiment_date = field_for(SequencingExperiment, 'experiment_date',
                                 field_class=DateOrDatetime)
+    umi_barcode_offset = field_for(SequencingExperiment, 'umi_barcode_offset',
+                                   fvalidate=validate_positive_number)
```

Comment:
Looks like there  is a typo in all of these? 
```suggestion
                                   validate=validate_positive_number)
```

Reply:
yeah. sorry about that, have updated them.